test/functional/ios/testapp/basics/calc-app-2-specs.js

```diff
@@ -74,9 +75,16 @@ describe('testapp - basics - calc app 2', function () {
       .log('crashlog').then(function (logsBefore) {
         numBeforeLogs = logsBefore.length;
         fs.writeFileSync(dir + '/myApp_' + Date.parse(new Date()) + '_rocksauce.crash', msg);
-      }).log('crashlog').then(function (logsAfter) {
+      })
+      .log('crashlog').then(function (logsAfter) {
         logsAfter.length.should.be.above(0);
-        logsAfter.length.should.not.equal(numBeforeLogs);
+        if (env.IOS83) {
+          // getting the crash logs in 8.3 consumes them, so we will only get the one added above
```

Comment:
More interesting, if my reading of [this](https://github.com/appium/appium/blob/master/lib/devices/ios/ios-crash-log.js#L56) is correct then why were things cumulative before? It seems like it ought to only send logs not yet read. 

Investigating.